CoulombsLaw/coulomb.py

```diff
@@ -0,0 +1,31 @@
+"""
+Coulomb's Law Calculations
+
+This module provides functionality to calculate the electric force between two point charges
+based on Coulomb's law.
+"""
+
+# Coulomb's constant in N·m²/C² (Coulomb's Law constant)
+COULOMBS_CONSTANT = 8.99e9
+
+# Function to calculate the electric force between two charges
+def calculate_electric_force(q1, q2, r):
+    """
+    Calculate the electric force between two point charges using Coulomb's law.
+
+    Args:
+    q1 (float): The first charge in coulombs (C).
```

Comment:
if you do not require a float here and do not check for it either I would only specify number in the docstring